pyopencl/cl/pyopencl-complex.h

```diff
@@ -274,11 +274,19 @@
     } \
   } \
 
+#if PYOPENCL_COMPLEX_ENABLE_EXTENDED_ALIGNMENT
```

Comment:
Add a comment that this is undocumented and not to be relied upon (i.e. may disappear at any time). I'm OK with proceeding like this, but I'm not totally sold that it's not a mistake. :slightly_smiling_face:

Reply:
Done in bea54874cd70661b05bf246596a289151235835f